### ISSUE TITLE
2025_2: BUG Experiments can not be renamed to a name that includes periods (".").

### ISSUE THREAD

**Issue by matthewtomo** [lzim/teampsd#3328]
- This is an issue, since periods are a valid character that are included in the auto-generated names.
- The error message could also be made more helpful to identify which character are valid (A-Z, 0-9, _, -, and .)
![image](https://github.com/user-attachments/assets/8caaad91-8f07-446b-aee5-ea157f7ef54b)
- @matthewtomo will handle this issue, and knows how to address it, but it is a lower priority for right now.

AC:
- [ ] assign to epic and sprint for next roadmap
- [ ] Update list of acceptable characters in regular expression test
- [ ] Update error message to be more useful

**Comment by lzim** [comment]
@matthewtomo 
Let's make sure any implementation of this is coordinated with #quant_workflow please to make sure we have compatibility across our programming language syntaxes.

I don't know why anything should be named or renamed with a period. 
Thx!

**Comment by lzim** [comment]
**wk3 2025_04 - _What is the update on this?_** 
Thanks! 

FYI: 
@jamesmrollins 
@matthewtomo
@ljmoody 
@lijenn

**Comment by lzim** [comment]
Great, thanks @matthewtomo 

>@lzim This bug was never assigned to a roadmap, epic, or sprint.
> If we wanted to develop the fix for this bug, it would be a small task I could address on my own.
> I would expect it to take an hour or less to resolve.

Thanks, if that's true, then let's get it done to avoid issues with valid names with periods.
I updated the card title and right hand clicks to reflect this. Please adjust if not correct.
Thanks again!

FYI: @ljmoody @jamesmrollins @lijenn

**Comment by lijenn** [comment]
wk3 2025_04
@matthewtomo
@ljmoody
@lijenn 

@matthewtomo will implement this fix and promote it Prod when ready.

**Comment by lzim** [comment]
Thanks
@matthewtomo
@ljmoody
@lijenn

Looks like this won't be completed this week and the details needed to be updated again for asynchronous tracking and planning across workflows - _will you please update?_
- [ ] title 
- [ ] align the 9 clicks on the right reflect current info

Thx!

> wk3 2025_04
@matthewtomo
@ljmoody
@lijenn

> @matthewtomo will implement this fix and promote it Prod when ready.

**Comment by lzim** [comment]
@matthewtomo - _Is this resolved?_ Thanks!

**Comment by matthewtomo** [comment]
> [@matthewtomo](https://github.com/matthewtomo) - _Is this resolved?_ Thanks!

@lzim, no, this fix was deprioritized to focus on the ["Q/H/F/D text not loading" ANODN](https://github.com/lzim/research/issues/1048).

If desired, I could implement this fix with effort <= 1 story point

**Comment by lzim** [comment]
@lijenn - _Will you please determine how to prioritize?_
_Do we need this?_ 

If no, let's close the card. Thx!
